tedana/stats.py

```diff
@@ -74,10 +74,15 @@ def computefeats2(data, mmix, mask=None, normalize=True):
     # demean masked data
     if mask is not None:
         data = data[mask, ...]
+    # normalize data (minus mean and divide by std) in the last dimension
+    # so that least-squares estimates represent correlation values (data_R)
+    # assuming mmix are also normalized
```

Comment:
This is a bug, in my opinion. Would you mind adding something to normalize the mixing matrix here? I think that the following should work appropriately:

```python
mmix = stats.zscore(mmix, axis=0)
```